model/src/main/java/org/cbioportal/model/GenericAssayEnrichment.java

```diff
@@ -38,4 +41,14 @@ public void setGenericEntityMetaProperties(HashMap<String, String> genericEntity
 		this.genericEntityMetaProperties = genericEntityMetaProperties;
 	}
 
+    public static int compare(GenericAssayEnrichment c1, GenericAssayEnrichment c2) {
+        return c1.getpValue().compareTo(c2.getpValue());
+    }
```

Comment:
Could you add a new line between the two methods? Also maybe better to put compare method after `setqValue` method.